rpm: Stop skipping scp tests (#714)
ansible/pylibssh: docs/changelog-fragments/714.packaging.rst

> Removed needless skipping of scp tests as they seem to work ok now -- by :user:`Jakuje`.

Reviewer: ```suggestion
Stopped skipping SCP tests in the RPM spec -- by :user:`Jakuje`.
```

Author: Reworded. Thanks!
